players.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Top World Players</title>
+	<h1>Winners of the WYYC 2016</h1>
+</head>
+<body>
+	<h2>Mens Finals</h2>
+<fieldset>
+	<ol>
+		<li>Shion Araya</li>
+		<li>Yamata Murata</li>
+		<li>Zach Gormley</li>
+		<li>Gentry Stein</li>
+	</ol>
+</fieldset>
+	<h3>Womens Finals</h3>
+<fieldset>
+	<ol>
+		<li>Beatriz Gallegos Gracia</li>
+		<li>Tessa Piccillo</li>
+		<li>Stephanie Haight</li>
+		<li>Chloe Monsonego</li>
+	</ol>
+</fieldset>
+<form>
+	<fieldset>
+		<div>
+			 <label>Who is your Favorite?:</label>
+      <select name="player">
+        <option value="thrower">Shion Araya</option>
+        <option value="thrower">Yamata Murata</option>
+        <option value="thrower">Zach Gormley</option>
+        <option value="thrower">Gentry Stein</option>
+        <option value="thrower">Beatriz Gallegos Gracia</option>
+        <option value="thrower">Tessa Piccillo</option>
+        <option value="thrower">Stephanie Haight</option>
+        <option value="thrower">Chloe Monsonego</option>
+      </select>
+		</div>
+	</fieldset>

```

Comment:
Your form should have a submit button.  Also, the make sure form method is a `get` not a `post` at this time.